lang/abcl/Portfile

```diff
@@ -26,11 +26,11 @@ platforms           darwin
 master_sites        https://abcl.org/releases/${version}/
 distname            abcl-src-${version}
 
-checksums           md5     433c4e793e4378a4e27aa34906c70dbb \
-                    sha1    6ae7b20c844e5c357169e3ecc9abe412db85ba0c \
-                    rmd160  138ebc4e9de31fce98b056669a9905d6d95158c2 \
-                    sha256  d51014b2be6ecb5bcaaacda0adf4607a995dd4b6e9e509c8a1f5a998b7649227 \
-                    size    2557628
+checksums           md5     9bbdf32caa0ed33f89c065fe60d1a989 \
+                    sha1    c2851f214181a2d3f8a68bd88c18624fd101864f \
+                    rmd160  24076d5f6539dcf759114530966a6d3acfa2a252 \
```

Comment:
`md5` and `sha1` checksums are deprecated.  You can remove those, and just keep `rmd160`, `sha256` and `size`.

Reply:
ACK.

I would suggest that the value returned via `port checksum -v` be suitably modified:  I essentially copy and paste, verify locally, and then put the paste in the Portfile.  I'll try to remember next time.